aspnetcore/performance/caching/memory/sample/WebCache/project.json

```diff
@@ -0,0 +1,49 @@
+﻿{
+  "dependencies": {
+    "Microsoft.AspNetCore.Diagnostics": "1.2.0-preview1-22808",
+    "Microsoft.AspNetCore.Mvc": "1.0.1",
+    "Microsoft.AspNetCore.Routing": "1.0.1",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.0.1",
+    "Microsoft.Extensions.Caching.Memory": "1.1.0",
+    "Microsoft.NETCore.App": {
+      "version": "1.0.1",
+      "type": "platform"
+    }
+  },
+
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
```

Comment:
imports shouldn't be needed anymore. Try removing them, if something doesn't work, we should figure out what the problem is.